pkg/apis/kubekey/v1alpha1/default.go

```diff
@@ -126,3 +126,35 @@ func SetDefaultClusterCfg(cfg *ClusterSpec) Kubernetes {
 
 	return defaultClusterCfg
 }
+
+func SetDefaultStorageCfg(cfg *ClusterSpec) Storage {
+	if cfg.Storage.LocalVolume.StorageClassName != "" {
+		cfg.Storage.LocalVolume.Enabled = true
+	}
+	if cfg.Storage.NfsClient.StorageClassName != "" {
+		cfg.Storage.NfsClient.Enabled = true
+	}
+	if cfg.Storage.CephRBD.StorageClassName != "" {
+		cfg.Storage.CephRBD.Enabled = true
+	}
+	if cfg.Storage.GlusterFS.StorageClassName != "" {
+		cfg.Storage.GlusterFS.Enabled = true
+	}
+
+	if cfg.Storage.DefaultStorageClass != "" {
```

Comment:
if **cfg.storage.DefaultStorageClass** is none of the listed cases,  what will happen ?

Reply:
No default storage classe will be set.